examples/llama.py

```diff
@@ -143,14 +143,7 @@ def __call__(self, tokens:Tensor, start_pos:int):
 
     # get only the part we are using. making it contiguous avoids more kernel calls
     freqs_cis = self.freqs_cis[:, start_pos:start_pos+seqlen].contiguous().realize()
-
-    if seqlen > 1:
-      mask = np.full((1, 1, seqlen, start_pos + seqlen), float("-inf"), dtype=np.float32)
-      mask = np.triu(mask, k=start_pos + 1)  # TODO: this is hard to do in tinygrad
-      mask = Tensor(mask)
-    else:
-      mask = None
-
+    mask = Tensor.full((1, 1, seqlen, start_pos + seqlen), float("-inf"), dtype=dtypes.float32).triu(start_pos+1) if seqlen > 1 else None
```

Comment:
did you check it works? `.where` is broken for `inf` values right now

Reply:
didnt run into any issues when running llama or stable diffusion after the change

Reply:
well, you should check the tensor has correct values, and not `nan`s

kapi from discord has `inf`s solution for `triu` and `tril`, but not general for `where`

Reply:
loading the weights runs without failing, but inference runs into 
ValueError: probabilities contain NaN